go/consensus/tendermint/tendermint.go

```diff
@@ -76,10 +76,9 @@ const (
 	cfgABCIPruneStrategy = "tendermint.abci.prune.strategy"
 	cfgABCIPruneNumKept  = "tendermint.abci.prune.num_kept"
 
-	// CfgP2PPrivatePeerID configures tendermint's private peer ID(s).
-	CfgP2PPrivatePeerID = "tendermint.private_peer_id"
-	// CfgP2PPersistentPeer configures tendermint's persistent peer(s).
-	CfgP2PPersistentPeer = "tendermint.persistent_peer"
```

Comment:
Hm, actually having an option to configure persistent peers may still be useful (e.g., to establish connections to known good peers).

Reply:
yeah added it back, so now i'm also thinking if there's even any benefit in setting `private_peer_ids` on the sentry node in our case. In tendermint's case it's useful since it prevents others from knowing which nodes are behind a specific sentry. In our case this isn't really true, since this will be visible from the registry. 

This would mean that there's no benefit of the (just added) `tendermint.sentry.upstream_address` flag, since it would do the same as the `persistent_peers` flag. So in that case, the sentry node configuration would be:

- on sentry node:
  - set sentry nodes as private_peers
- on upstream node:
  - `worker.registration.sentry.address`, `worker.registration.sentry.cert_fiile` (for querying addresses at registration time) (just renames of the existing flags)
  - `tendermint.disable_peer_exchange`

^ which seems simple enough